src/components/SelectionList/Search/ActionCell.tsx

```diff
@@ -53,6 +53,8 @@ function ActionCell({onButtonPress, action = CONST.SEARCH.ACTION_TYPES.VIEW, isL
             small
             pressOnEnter
             style={[styles.w100]}
+            innerStyles={[styles.bgTransparent]}
+            textStyles={[styles.link]}
```

Comment:
I think rather than styling it this way, we should create a new prop to pass in the same way we might pass success or danger. Maybe we can simply pass in "link" and then add the styles from there.